### PR TITLE
Add react-native-fs

### PR DESCRIPTION
**Pull request by pocesar**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Run `tsc` without errors.
- [x] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

### CODE REVIEW

File: react-native-fs/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+declare namespace RNFS {
+  interface ReadDirItem {
+    // The name of the item
+    name: string;
+    // The absolute path to the item
+    path: string;
+    // Size in bytes
+    size: string;
+    // Is the file just a file?
+    isFile: () => boolean;
```
Comment (andy-ms) [inline react-native-fs/index.d.ts]: Could this be `isFile(): boolean`?

**Review by ghost — COMMENTED** [review]
Missing `tslint.json` (should contain `{ "extends": "../tslint.json" }`)

File: react-native-fs/index.d.ts (hunk 1 above)
Comment (pocesar) [inline react-native-fs/index.d.ts]: indeed, I copied their 'flow' typings from the readme

**Comment by pocesar** [comment]
done

**Pull request merged**